action-server/covidflow/jobs/send_reminders.py

```diff
@@ -64,7 +64,11 @@ def run(
         _send_reminders(reminders, hashids, endpoints)
     )
 
-    logger.info("Sent reminders", sent_reminders=sent, errored_reminders=errored)
+    logger.info(
+        f"Sent reminders: {len(sent)} sent, {len(errored)} errored",
```

Comment:
Maybe it's a little confusing to have the number of reminders in the string but reminder ids as attributes.

Reply:
we could have `x reminder sent, x errored`